api/specs/webserver/scripts/_common.py

```diff
@@ -63,7 +63,7 @@ def create_openapi_specs(app: FastAPI, file_path: Path):
     # Removes default response 422
     for _, method_item in openapi.get("paths", {}).items():
         for _, param in method_item.items():
-            param.get("responses", {}).pop("422")
+            param.get("responses", {}).pop("422", None)
```

Comment:
Q: Just for my sake, why do you need to do that here?

Reply:
By default fastapi adds 422 to all responses and I remove it here. But there is a bug if you want to explicitly add 422 ... will take care of that.